.github/workflows/master.yml

```diff
@@ -35,6 +35,7 @@ env:
   MVN_OPT: -Dmaven.javadoc.skip=true -Drat.skip=true -Dscalastyle.skip=true -Dspotless.check.skip -Dorg.slf4j.simpleLogger.defaultLogLevel=warn -Pjdbc-shaded -Dmaven.plugin.download.cache.path=/tmp/engine-archives
   KUBERNETES_VERSION: v1.26.1
   MINIKUBE_VERSION: v1.29.0
+  "aether.connector.connectTimeout": 60000
```

Comment:
I am not familiar with this property, I want to know if it can be placed in `MVN_OPT` as `-Daether.connector.connectTimeout=60000` or can it be configured in `pom.xml`? thanks  ~

Reply:
Not sure about this. All I found is "To set one of the configuration options from above just use system variables. " mentioned in the description of this PR. But as you see some tests still failed when resolving dependencies, it seems that setting it here in the system env variable does not help. I would like to see what happens when you are doing setting it via property in pom in Spark.

Reply:
introduce a `settings.xml` looks too complex for new developers